mappings/com/mojang/blaze3d/systems/RenderSystem.mapping

```diff
@@ -329,6 +329,7 @@ CLASS com/mojang/blaze3d/systems/RenderSystem com/mojang/blaze3d/systems/RenderS
 	METHOD setupGuiFlatDiffuseLighting setupGuiFlatDiffuseLighting (Lorg/joml/Vector3f;Lorg/joml/Vector3f;)V
 		ARG 0 light0
 		ARG 1 light1
+	METHOD setupLevelDiffuseLighting setupLevelDiffuseLighting (Lorg/joml/Vector3f;Lorg/joml/Vector3f;)V
```

Comment:
raaaaaaaaaaaaaaah I don't like level. since it's in render system we can leave this as is